.github/workflows/release.yaml

```diff
@@ -28,16 +28,18 @@ jobs:
         with:
           fetch-depth: 0
       - name: Get changed files
-        id: changed-files
+        id: changed-markdown-files
         uses: tj-actions/changed-files@800a2825992141ddde1a8bca8ad394cec34d3188 # tag=v42.0.5
+        with:
+          files: CHANGELOG/**.md
       - name: Get release version
         id: release-version
         run: |
-            if [[ ${{ steps.changed-files.outputs.all_changed_files_count }} != 1 ]]; then
-              echo "1 release notes file should be changed to create a release tag, found ${{ steps.changed-files.outputs.all_changed_files_count }}"
+            if [[ ${{ steps.changed-markdown-files.outputs.all_changed_files_count }} != 1 ]]; then
+              echo "1 release notes file should be changed to create a release tag, found ${{ steps.changed-markdown-files.outputs.all_changed_files_count }}"
```

Comment:
After we get this PR merged, it looks like the changed-files action has a built-in way to feed the list of files into a matrix that can spawn one job per file which might be able to make cutting multiple patches pretty slick.

https://github.com/tj-actions/changed-files/blob/257d47dfba22be3e0a17f6bad47ff07f7e76747c/.github/workflows/matrix-example.yml